docs/elements/dropbox/tips.md

```diff
@@ -11,15 +11,21 @@ order: 50
 sitemap: false
 ---
 
-### Tips
+# Tips
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
 
-__How can I test out the new Dropbox V2 Element?__
+{% include callout.html content="<strong>On this page</strong><br/><a href=#general>General</a><br/><a href=#dropbox V2>Dropbox V2</a>" type="info" %}
 
-Dropbox V2 API is now supported within the Cloud Elements Platform.  We want to give customers a chance to test the Dropbox V2 Element within their application.  To access the new Element, please create an Instance using the key `dropboxv2`.  Please let [support](mailto:support@cloud-elements.com) know if you have any questions.
 
-__What new features does the Dropbox V2 Element include within the Cloud Elements Platform?__
+## General
 
-The V2 Element includes a `refId` for certain APIs.  This is the actual ID of the file sent back from Dropbox.  It can be used as the `ID` with all API calls with the exception of `GET /folders/{id}/contents`.
+* Dropbox files that have not been permanently deleted will still exist in your Trash folder and will generate an `UPDATED` event type if they are recreated.
+
+## Dropbox V2
+
+* Dropbox V2 API is now supported within the Cloud Elements Platform.  We want to give customers a chance to test the Dropbox V2 Element within their application.  To access the new Element, please create an Instance using the key `dropboxv2`.  
```

Comment:
This doesn't look correct.  There's only one Dropbox element and there isn't a v2 element key.